inc/functions/formatting.php

```diff
@@ -102,6 +102,25 @@ function rocket_validate_js( $file ) {
 	return sanitize_text_field( \rocket_remove_url_protocol( strtok( $file, '?' ) ) );
 }
 
+/**
+ * Sanitize and validate CSS files to exclude from the minification.
+ *
+ * @since  3.7
+ *
+ * @param  string $file filepath to sanitize.
+ * @return string
+ */
+function rocket_validate_css( $file ) {
+	if ( rocket_is_internal_file( $file ) ) {
+		$file = trim( $file );
+		$file = rocket_clean_exclude_file( $file );
+
+		return $file;
```

Comment:
Let's combine these to be more concise and performant.

```suggestion
		return rocket_clean_exclude_file( trim( $file ) );
```